src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/utils/lambda_constants.py

```diff
@@ -0,0 +1,7 @@
+CIRCLECI_CONFIG_SECRET_ENV = "CIRCLECI_CONFIG_SECRET"
```

Comment:
Add a comment enumerating the places these are hardcoded (e.g., the Lambda rotator function)

Reply:
Added